### ISSUE TITLE
Testnet halted at 2018015

### ISSUE THREAD

**Issue by jun0tpyrc** [litecoin-project/litecoin#745]
https://blockexplorer.one/litecoin/testnet 

2018015 | 1,000 | 4 | 33bb3fd5f4ebc1d9cf6e3f70362870aa7bc0ba185432ae1396b84ec27520a40c | 12.5 LTC | 01/09/2021 - 10:34:17 UTC1 day ago
-- | -- | -- | -- | -- | --

same as private hosted node running on litecoin:0.17.1

2021-09-03T05:24:14Z Potential stale tip detected, will try using extra outbound peer (last tip update: 154196 seconds ago)
2021-09-03T05:24:14Z New outbound peer connected: version: 70015, blocks=2018015, peer=568962

**Comment by SomethingUseful** [comment]
same exact problem here. tried delete peers and reindex and it still stopped at same block. its broken.

**Comment by losh11** [comment]
I'm currently attempting to fix this.

**Comment by losh11** [comment]
@jun0tpyrc @SomethingUseful testnet is back to mining blocks!

**Comment by gituser** [comment]
Litecoin testnet is stuck again, now at [2,263,967] block

https://chain.so/testnet/ltc
https://blockexplorer.one/litecoin/testnet/

@losh11 maybe you can fix this again?

**Comment by losh11** [comment]
@gituser the issue you are facing is unrelated. With the activation of MWEB on Litecoin Testnet, all nodes must update to the 0.21.2rc5 build to receive data from nodes. All users running a Litecoin Testnet node are advised to update ASAP, binaries for rc5 are available from dev.litecoin.org.

You may also need to manually add a 0.21.2 node, which can be done from console as shown below:

`litecoin-cli addnode 78.62.46.195:19333`

A future build will include updated fixed peers including 0.21.2 nodes so there won’t be a need to add a 0.21 node manually.

**Comment by jun0tpyrc** [comment]
> @gituser the issue you are facing is unrelated. With the activation of MWEB on Litecoin Testnet, all nodes must update to the 0.21.2rc5 build to receive data from nodes. All users running a Litecoin Testnet node are advised to update ASAP, binaries for rc5 are available from dev.litecoin.org.
> 
> You may also need to manually add a 0.21.2 node, which can be done from console as shown below:
> 
> `litecoin-cli addnode 78.62.46.195:19333`
> 
> A future build will include updated fixed peers including 0.21.2 nodes so there won’t be a need to add a 0.21 node manually.

could you please provide peer that survive after the fork?
can't ingest new blocks after some time , i can confirm it already being added
```
litecoin-cli --chain=test --rpcuser=XXXX -rpcpassword=XXXXX addnode "78.62.46.195:19333" add
error code: -23
error message:
Error: Node already added
```

and i rollbacked to older blocks to make sure my node doesn't go the wrong network , but still can't resume
```
  "blockId": "6aa051b99b65088283fd659d948f96de88868d5ac563db258a9fe63eb0d9280d",
  "blockTime": "2022-03-30T00:22:07.000Z",
```
